etc/tikv.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=TiKV, a distributed transactional key value store.
+Wants=network-online.target
+
+[Service]
+User=tikv
+ExecStart=/usr/bin/tikv-server --config $CONFIGURATION_DIRECTORY/config.toml
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=on-failure
+KillSignal=SIGINT
+LimitNOFILE=82920
```

Comment:
By default, nearly all Linux distribution uses 1024 for max number of file descriptors. I'd recommend making the default configuration with `max-open-files` to 1024, so users can start tikv without any problems when they install the package. And we can have a tuning guide to tell user to modify system `ulimit` settings.

Reply:
Agree, should we change the default in `/etc/config-template.toml`, or create a config specifically for packages?

Reply:
1024 is the default value

Reply:
Sadly it isn't:

https://github.com/tikv/tikv/blob/535ac80b66050c3d624838719cc65fe850ee5f9e/etc/config-template.toml#L348
https://github.com/tikv/tikv/blob/535ac80b66050c3d624838719cc65fe850ee5f9e/etc/config-template.toml#L348

Reply:
@tennix @sykp241095 do you have further guidance on this?